go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/prometheus/client_golang
+
+require (
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973
+	github.com/gogo/protobuf v1.1.1
```

Comment:
Inclusion of this happens via prometheus/wrap_test.go, but it's an accident from my side.
I'll send you a PR to fix, and then this should be updated to not include this line.